src/App/Model/CryptAES.php

```diff
@@ -37,7 +37,7 @@ public function decrypt($crypt)
     {
         if (!$this->cipher && !$this->setCipher()) return $crypt;
 
-        return $this->cipher->decrypt($crypt);
+        return $this->cipher->decrypt(base64_decode($crypt));
```

Comment:
I had problems to write the encoded data to the mysql-database. So I have uses this dirty tweak. Can you please try it with your mysql-Version?